src/display.rs

```diff
@@ -285,28 +283,37 @@ impl Display {
         // events into one.
         let mut new_size = None;
 
+        // Update the DPR 
+        let dpr = self.window.hidpi_factor();
+
```

Comment:
There's a trailing whitespace after the "DPR" comment here and it's probably cleaner to have just one empty line instead of two after `let dpr = ...;`.